src/devices/tfa_14_1504_v2.c

```diff
@@ -0,0 +1,141 @@
+/** @file
+    Decoder for TFA Dostmann 14.1504.V2 (30.3254.01)
+    Radio-controlled grill and meat thermometer
+
+    Copyright (C) 2022-2023 Joël Bourquard <joel.bourquard@gmail.com>
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+
+/**
+Decoder for TFA Dostmann 14.1504.V2 (30.3254.01)
+
+CAUTION: Do not confuse with TFA Dostmann 14.1504 (30.3201) which had a completely different protocol => [71] Maverick ET-732/733 BBQ Sensor
+
+Payload format:
+- Preamble         {36} 0x7aaaaaa5c (for robustness we only use the tail: {24}0xaaaa5c)
+- Flags            {4}  OR between: 0x2=battery ok, 0x5=resync button
+- Temperature      {12} Raw temperature value. Temperature in ºC = (value/4)-532. Example: 0x8a0 = 20 ºC
+- Separator        {8}  0xff (differs if resync)
+- Digest           {16} 16-bit LFSR digest + final XOR
+
+To get raw data:
+```
+rtl_433 -R 0 -X 'n=TFA-141504v2,m=FSK_PCM,s=360,l=360,r=4096,preamble={24}aaaa5c'
+```
+
+Example payloads (excluding preamble):
+- Resync   = 7052f9cee3 (encoding differs from temperature readings => not handled)
+- No probe = 2700ffb791 (just like a temperature reading => in this case we report the appropriate probe status and no temperature reading)
+- ...
+- 20 ºC    = 28a0ffce69
+- 21 ºC    = 28a4ffa0f5
+- ...
+- 24 ºC    = 28b0ff6438
+- ...
+- 44 ºC    = 2900ff8c9d
+- ...
+*/
+
+#include "decoder.h"
+
+#define NUM_BITS_PREAMBLE 24
+#define NUM_BYTES_DATA    5
+#define OFFSET_MIC        (NUM_BYTES_DATA - 2)
+#define NUM_BITS_DATA     (NUM_BYTES_DATA * 8)
+#define NUM_BITS_TOTAL    (NUM_BITS_PREAMBLE + NUM_BITS_DATA)
+#define NUM_BITS_MAX      (NUM_BITS_TOTAL + 12)
+
+static int tfa_14_1504_v2_callback(r_device *decoder, bitbuffer_t *bitbuffer)
+{
+    uint8_t const preamble[] = {
+            0xaa,
+            0xaa,
+            0x5c,
```

Comment:
Less lines if all bytes are in a row.

Reply:
Hi @merbanan , thank you!   Change applied: https://github.com/merbanan/rtl_433/pull/2296/commits/d145440875ebbc91f26b5ceb329072f81a18f51a